alchemy/bin/commands/configure.ts

```diff
@@ -0,0 +1,194 @@
+import {
+  confirm,
+  group,
+  intro,
+  isCancel,
+  log,
+  multiselect,
+  outro,
+  password,
+  select,
+  text,
+} from "@clack/prompts";
+import pc from "picocolors";
+import z from "zod";
+import { Credentials, Profile, Provider } from "../../src/auth.ts";
+import { CloudflareAuth } from "../../src/cloudflare/auth.ts";
+import { listCloudflareAccounts } from "../../src/cloudflare/user.ts";
+import { authProcedure, CancelSignal } from "../trpc.ts";
+import { cloudflareLogin } from "./login.ts";
+
+export const configure = authProcedure
+  .meta({
+    description: "configure a profile",
+  })
+  .input(
+    z.object({
+      profile: z
+        .string()
+        .optional()
+        .meta({ alias: "p" })
+        .describe("the profile to configure"),
+    }),
+  )
+  .mutation(async ({ input }) => {
+    intro(pc.cyan("🧪 Configure Profile"));
+    const name = await promptForProfileName(input);
+    const profile = await Profile.get(name);
+    if (profile) {
+      log.info(`Profile: ${pc.bold(name)}`);
+      for (const [provider, providerProfile] of Object.entries(profile)) {
+        const description = [
+          `- ${pc.bold(provider)}: ${providerProfile.metadata.name} (${pc.dim(providerProfile.metadata.id)})`,
+          `  - Method: ${pc.dim(providerProfile.method)}`,
+          ...(providerProfile.scopes?.length
+            ? [`  - Scopes: ${pc.dim(providerProfile.scopes.join(", "))}`]
+            : []),
+        ];
+        log.info(description.join("\n"));
+      }
+      if (
+        (await confirm({
+          message: `Update profile ${pc.bold(name)}?`,
+          initialValue: false,
+        })) !== true
+      ) {
+        throw new CancelSignal();
+      }
+    }
+    const method = await select({
+      message: `Select a login method for ${pc.bold("Cloudflare")}`,
+      options: [
+        { label: "OAuth", value: "oauth", hint: "Recommended" },
+        { label: "API Token", value: "api-token" },
+        { label: "API Key", value: "api-key", hint: "Legacy" },
```

Comment:
Should we update this to `Global API Key`, I think a lot of people will get confused by API Token vs Key